scripts/api-documentation/api-doc-config.json

```diff
@@ -0,0 +1 @@
rverPath  }}/ZZA","_type":"request"},{"_id":"env_b30c61d7d0a73c30613e4f83449d1df1cb77c75a","color":null,"created":1590743470787,"data":{"graphqlServer":{"authorization":"","path":"{{ protocol  }}://{{ graphQLServerDomain  }}"},"webComponent":{"appointmentsPath":"{{ protocol  }}://{{ webComponentServerDomain  }}","configServerHealthPath":"{{ protocol  }}://{{ webComponentConfigServerDomain  }}/dev/v1/health","configServerPath":"{{ protocol }}://{{ webComponentConfigServerDomain  }}/dev/v1/web-components-config","searchWidgetPath":"{{ protocol  }}://{{ webComponentServerDomain  }}","token":""}},"dataPropertyOrder":{"&":["webComponent","graphqlServer"],"&~|graphqlServer":["path","authorization"],"&~|webComponent":["configServerPath","configServerHealthPath","searchWidgetPath","appointmentsPath","token"]},"isPrivate":false,"metaSortKey":1590743470787,"modified":1591072302154,"name":"Base Environment","parentId":"wrk_88e31d48b55f412b995f74e737d9ee75","_type":"environment"},{"_id":"jar_b30c61d7d0a73c30613e4f83449d1df1cb77c75a","cookies":[],"created":1590743470796,"modified":1590743470796,"name":"Default Jar","parentId":"wrk_88e31d48b55f412b995f74e737d9ee75","_type":"cookie_jar"},{"_id":"env_91c1ac7e91254caaa5fd4d89f33a6bef","color":"#ecec00","created":1590743508931,"data":{"graphQLServerDomain":"8g8ibl8iei.execute-api.eu-west-2.amazonaws.com/dev/v1/graphql","protocol":"https","webComponentConfigServerDomain":"vwvdddbw9k.execute-api.eu-west-2.amazonaws.com","webComponentServerDomain":"b2q5q0tc48.execute-api.eu-west-2.amazonaws.com"},"dataPropertyOrder":{"&":["protocol","webComponentConfigServerDomain","webComponentServerDomain","graphQLServerDomain"]},"isPrivate":false,"metaSortKey":1,"modified":1591070195443,"name":"development","parentId":"env_b30c61d7d0a73c30613e4f83449d1df1cb77c75a","_type":"environment"},{"_id":"env_26e0603a03b04ea0b9193141c364e50a","color":"#00c400","created":1590743650077,"data":{"graphQLServerDomain":"localhost:4000","protocol":"http","webComponentConfigServerDomain":"localhost:3000","webComponentServerDomain":"localhost:3000"},"dataPropertyOrder":{"&":["protocol","webComponentConfigServerDomain","webComponentServerDomain","graphQLServerDomain"]},"isPrivate":false,"metaSortKey":0,"modified":1591030760062,"name":"local","parentId":"env_b30c61d7d0a73c30613e4f83449d1df1cb77c75a","_type":"environment"},{"_id":"env_d301a83fdd3340ebb667ae00304ba8b6","color":"#d00000","created":1590748587252,"data":{"graphQLServerDomain":"8g8ibl8iei.execute-api.eu-west-2.amazonaws.com/dev/v1/graphql","protocol":"https","webComponentConfigServerDomain":"vwvdddbw9k.execute-api.eu-west-2.amazonaws.com","webComponentServerDomain":"b2q5q0tc48.execute-api.eu-west-2.amazonaws.com"},"dataPropertyOrder":{"&":["protocol","webComponentConfigServerDomain","webComponentServerDomain","graphQLServerDomain"]},"isPrivate":false,"metaSortKey":2,"modified":1591070228389,"name":"production","parentId":"env_b30c61d7d0a73c30613e4f83449d1df1cb77c75a","_type":"environment"}]}
```

Comment:
Is there any chance you can use prettier to format the JSON before committing - will help with version control